src/main/java/com/revature/chronicle/controller/FirebaseController.java

```diff
@@ -70,13 +109,35 @@ public JSONArray getUsers() throws FirebaseAuthException {
 	 */
```

Comment:
I do not believe anyone gave me an answer for why you want both register methods here. Are you using both in the application?